modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_compare_runs_card.scss

```
@@ -0,0 +1,41 @@
+.compare-runs-floating-box {
+	background-color: white;
+	top: 213px;
+	box-shadow: 0 2px 15px 0 rgba(0, 0, 0, 0.04),
+		0 4px 14px 0 rgba(0, 0, 0, 0.1);
+	margin: auto;
+	position: absolute;
+	width: 490px;
+}
```

Correct alphabetical order


```suggestion
.compare-runs-floating-box {
	background-color: white;	
	box-shadow: 0 2px 15px 0 rgba(0, 0, 0, 0.04),
		0 4px 14px 0 rgba(0, 0, 0, 0.1);
	margin: auto;
	position: absolute;
	top: 213px;
	width: 490px;
}
```